### PR TITLE
buildozer: make ellipsis work with BUILD.bazel and BUCK files

### LINKED ISSUE #91: buildozer: ellipsis doesn't find BUILD.bazel or BUCK files
For example, in the buildtools repo:
```
$ bazel-bin/buildozer/buildozer '//edit/...:*' "print"
$ cp edit/BUILD.bazel edit/BUILD
$ bazel-bin/buildozer/buildozer '//edit/...:*' "print"
rule "//edit/...:" has no attribute "name"
(missing) load
go_default_library go_library
go_default_test go_test
$
```

### PR DESCRIPTION
**Pull request by swolchok**
Fixes #91.

Regarding CLA, this contribution is being made on behalf of my employer, Facebook.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: edit/buildozer.go [hunk 1]
```diff
@@ -692,8 +700,13 @@ func rewrite(commandsForFile commandsForFile) *rewriteResult {
 		}
 	} else {
 		origName := name
-		name = strings.TrimSuffix(name, "BUILD")
-		for _, suffix := range buildFileSuffixes {
+		for suffix := range buildFileNames {
```
Comment (pmbethe09) [inline edit/buildozer.go]: the problem with this change is the intentionally random iteration order of Go maps.

You probably need to keep the list for this part and the map for below

Comment (swolchok) [inline edit/buildozer.go]: good catch, I forgot that the iteration order mattered.

**Comment by swolchok** [comment]
should be fixed!

**Comment by pmbethe09** [comment]
Jenkins, test this please.

**Comment by pmbethe09** [comment]
LGTM, just waiting for CI

**Pull request merged**